### PR TITLE
[DOC] update PixArt README

### PR DESCRIPTION
**Pull request by zhtmike**
# What does this PR do?

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/pixart_sigma/README.md [hunk 1]
```diff
@@ -165,20 +171,19 @@ Followed by some generated images using the testing prompts.
...
+| PixArt-Sigma | 4     | 512 (multi)  | 3~5 mins      | 32         | ON        | OFF       | O1        | 3.817s    | 33.5          | [pixart-sigma-512-MS.yaml](configs/train/pixart-sigma-512-MS.yaml)   |
+| PixArt-Sigma | 4     | 1024 (multi) | 3~5 mins      | 12         | ON        | OFF       | O1        | 7.042s    | 6.8           | [pixart-sigma-1024-MS.yaml](configs/train/pixart-sigma-1024-MS.yaml) |
+| PixArt-Sigma | 4     | 2048 (multi) | 3~5 mins      | 1          | ON        | OFF       | O1        | 8.772s    | 0.5           | [pixart-sigma-2K-MS.yaml](configs/train/pixart-sigma-2K-MS.yaml)     |
 
-> Context: {Ascend chip}-{number of NPUs}-{mindspore version}\
-> Bucket Training: Training images with different aspect ratios based on bucketing.\
-> VAE/T5 Cache: Use the pre-generated T5 Embedding and VAE Cache for training.\
-> Speed (step/s): sampling speed measured in the number of training steps per second.\
-> FPS (img/s): images per second during training. average training time (s/step) = global batch_size / FPS
+> step time: training time measured in the number of seconds for each training stesp.\
```
Comment (CaitinZhao) [inline examples/pixart_sigma/README.md:185]:
```suggestion
> step time: training time measured in the number of seconds for each training steps.\
```

**Review by CaitinZhao — APPROVED** [review]

File: examples/pixart_sigma/README.md (hunk 1 above)
Comment (zhtmike) [reply examples/pixart_sigma/README.md]: done, thx

**Review by SamitHuang — APPROVED** [review]

**Comment by SamitHuang** [comment]
inference performance table to be updated

**Comment by zhtmike** [comment]
> inference performance table to be updated

done.

**Pull request merged**